### DISCUSSION TITLE
Content Preview is not showing after content upload

### DISCUSSION THREAD

**Discussion by devaj04** [Sunbird-coKreat/Community#79] [Bugs]
Content Preview is not showing after content upload
contentID: do_31378807165662003212263

URL - https://obj.diksha.gov.in/ntp-content-production/content/assets/do_31378807165662003212263/busra-1-0224.pdf?45
if remove ?45 in the URL then PDF is showing

I have attached one mail screenshot for your reference

Please find the following Jira ticket for your reference:
https://project-diksha.atlassian.net/browse/DU-541

we are facing all types of content while content upload on the portal https://diksha.gov.in/workspace/content/edit/generic#no

![image](https://user-images.githubusercontent.com/125238862/236147349-1e04cc46-75c5-482a-ae4c-70bb1dda9bef.png)
![image](https://user-images.githubusercontent.com/125238862/236147407-1ca2a8ae-5e27-41a3-8316-c80e80dd05be.png)
![image](https://user-images.githubusercontent.com/125238862/236147452-025c326e-9e14-43db-bcbd-46ddaa5df413.png)
![image](https://user-images.githubusercontent.com/125238862/236147473-4016f953-0da5-46d5-a772-8f3986b4e248.png)
![image](https://user-images.githubusercontent.com/125238862/236147495-c1eea37e-9033-410e-9ed0-c1b0bece3ee3.png)
![image](https://user-images.githubusercontent.com/125238862/236147508-91e82ca6-75a8-4a74-89d2-93bf10c16806.png)

**Comment by devaj04** [comment]
Earlier it was raised in ED
https://github.com/orgs/Sunbird-Ed/discussions/479

**Comment by mmanimegalai** [comment]
Please follow https://github.com/orgs/Sunbird-Ed/discussions/479 for updates on this issue

**Comment by devaj04** [comment]
Ok, following https://github.com/orgs/Sunbird-Ed/discussions/479
